d/handy/source/app.d

```diff
@@ -5,7 +5,7 @@ void main() {
     import slf4d;
     import slf4d.default_provider;
 
-    auto provider = new shared DefaultProvider(true, Levels.ERROR);
+    auto provider = new shared DefaultProvider(true, Levels.DEBUG);
```

Comment:
is there a way to catchup up compiler flag here @andrewlalis @cyrusmsk ?

I mean, if `-d` is release default value to **Levels.ERROR** and if `-d` is debub default value to **Levels.DEBUG**